x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/logs.tsx

```diff
@@ -70,18 +64,25 @@ const TabComponent = (props: TabProps) => {
           />
         </EuiFlexItem>
         <EuiFlexItem grow={false}>
-          <EuiButtonEmpty
-            data-test-subj="infraTabComponentOpenInLogsButton"
-            size={'xs'}
-            flush={'both'}
-            iconType={'popout'}
-            {...nodeLogsMenuItemLinkProps}
-          >
-            <FormattedMessage
-              id="xpack.infra.nodeDetails.logs.openLogsLink"
-              defaultMessage="Open in Logs"
-            />
-          </EuiButtonEmpty>
+          <RedirectAppLinks coreStart={services}>
+            <EuiButtonEmpty
+              data-test-subj="infraTabComponentOpenInLogsButton"
+              size={'xs'}
+              flush={'both'}
+              iconType={'popout'}
+              href={locators.nodeLogsLocator.getRedirectUrl({
```

Comment:
Does this need to be called on each render? (I wonder if this comes with a slight performance hit due to the Base64 encoding).